api/src/pcapi/routes/backoffice_v3/templates/accounts/get/details/personal_information_registration.html

```diff
@@ -1,112 +1,19 @@
-                  </span>
-                </p>
-              {% endif %}
-              <p class="d-flex justify-content-between">
-                <span class="fs-6">Statut</span>
-                {% if idCheckItem.status == 'ok' %}
-                  <span class="fs-6 badge rounded-pill text-bg-success align-middle px-3 py-2 fs-5">
-                    <i class="bi bi-check-circle"></i> {{ idCheckItem.status.upper() }}
-                  </span>
-                {% elif idCheckItem.status == 'ko' %}
-                  <span class="fs-6 badge rounded-pill text-bg-danger align-middle px-3 py-2 fs-5">
-                    <i class="bi bi-exclamation-circle"></i> {{ idCheckItem.status.upper() }}
-                  </span>
-                {% elif idCheckItem.status == 'suspicious' %}
-                  <span class="fs-6 badge rounded-pill text-bg-warning align-middle px-3 py-2 fs-5">
-                    <i class="bi bi-shield-shaded"></i> {{ idCheckItem.status.upper() }}
-                  </span>
-                {% elif idCheckItem.status %}
-                  <span class="fs-6 badge rounded-pill text-bg-warning align-middle px-3 py-2 fs-5">
-                    <i class="bi bi-exclamation-circle"></i> {{ idCheckItem.status.upper() }}
-                  </span>
-                {% endif %}
-              </p>
-              <p class="d-flex justify-content-between">
-                <span class="fs-6">Explication</span>
-                <span class="fs-6">{{ idCheckItem.reason | empty_string_if_null | pc_backoffice_public_account_link_in_comment | safe }}</span>
-              </p>
-              <p class="d-flex justify-content-between">
-                <span class="fs-6">Codes d'erreur</span>
-                <span class="fs-6">{{ idCheckItem.reasonCodes | format_string_list }}</span>
-              </p>
-            </div>
-            <div class="col">
-              <div>
-                <div class="col justify-content-between">
-                  <button class="btn btn-primary fs-6"
-                          type="button"
-                          data-bs-toggle="collapse"
-                          data-bs-target=".pc-collapse-details-{{ idCheckItem.thirdPartyId }}">
-                    Afficher les détails techniques
-                  </button>
-                </div>
-                <div class="col">
-                  <div class="fs-6 collapse pc-collapse-details-{{ idCheckItem.thirdPartyId }}">
-                    <pre>
-                                         <code><br>{{ idCheckItem.technicalDetails  | tojson(indent=4) | safe | empty_string_if_null }}</code>
-                                    </pre>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    {% endfor %}
-  </div>
-</div>
+{% from "components/public_accounts/registration_steps.html" import build_step_content with context %}
```

Comment:
```suggestion
{% import "components/public_accounts/registration_steps.html" as steps %}
```
Un seul `import` et ensuite tu préfixes par `steps.`